components/textfield/stories/textfield.stories.js

```diff
@@ -166,9 +185,25 @@ export default {
 				disable: true,
 			},
 		},
+		hasCharacterCount: {
+			name: "Has character count",
```

Comment:
Should we add a `description` at all? It does seem pretty self-explanatory to me, so feel free not to.